thrift/shared.thrift

```diff
@@ -96,6 +96,10 @@ exception ClientVersionNotSupportedError {
   3: required string supportedVersions
 }
 
+exception FeatureNotEnabledError {
```

Comment:
Shall we use this error somewhere?

Reply:
This is used in server code currently. We might want to expose this to the user when they are requesting a specific behavior.

Reply:
yeah this is only until the client side changes go in